docs/read_the_docs/source/workflow_inputs/create_characteristics_rst.rb

```diff
@@ -46,4 +79,44 @@ def to_underscore_case
   write_subsection(f, row, 'Created by', '*')
   write_subsection(f, row, 'Source', '*')
   write_subsection(f, row, 'Assumption', '*')
+
+  # Arguments
+  r_arguments = []
+  lookup_csv_data.each do |lookup_row|
+    next if lookup_row[0] != parameter
+    next if lookup_row[2] != 'ResStockArguments'
+
+    lookup_row[3..-1].each do |argument_value|
+      argument, _value = argument_value.split('=')
+      r_arguments << argument if !r_arguments.include?(argument)
+    end
+  end
+  next if r_arguments.empty?
+
+  name = 'Arguments'
+  f.puts(name)
+  f.puts('*' * name.size)
+  f.puts
+  f.puts('.. list-table::')
+  f.puts('   :header-rows: 1')
+  f.puts
+  f.puts('   * - Name')
+  f.puts('     - Units')
+  f.puts('     - Choices')
+  f.puts('     - Description')
+  r_arguments.sort.each do |r_argument|
```

Comment:
Why sort? Seems like it would be better to preserve the order in the measure. For example, it'd be nice if `clothes_washer_present` was the first clothes washer argument listed.

Reply:
The order in `r_arguments` is determined by the order (from left to right) in `options_lookup.tsv`. But I could probably sort them according to the order in the measure.xml...